consensus/safety-rules/src/tests/safety_rules.rs

```diff
@@ -2,13 +2,48 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use crate::{test_utils, tests::suite, SafetyRules, TSafetyRules};
-use libra_types::validator_signer::ValidatorSigner;
+use libra_types::{
+    epoch_state::EpochState, validator_signer::ValidatorSigner,
+    validator_verifier::ValidatorVerifier,
+};
 
 #[test]
 fn test() {
     suite::run_test_suite(safety_rules);
 }
 
+// Tests for fetching a missing validator key from persistent storage.
+#[test]
+#[should_panic(expected = "Unable to retrieve consensus private key")]
+fn test_key_not_in_store() {
```

Comment:
again should be an error so it can go back into the suite

Reply:
Fixed